source/source-connector/usage-examples/topic-naming.txt

```diff
@@ -113,14 +123,121 @@ mapping, the connector performs the following actions:
+topic namespace map. To use a regular expression, start the namespace
+pattern with the forward slash (``/``) character. Create the regular
+expression following the syntax and behavior specified by the
+``java.util.regex.Pattern`` class.
+
+.. important::
+   
+   Because the ``/`` character denotes that the namespace is a
+   regex pattern, the connector raises a ``ConnectConfigException``
+   if the namespace includes this character in a non-regex context.
+
+The connector computes the topic name by doing variable expansion on the
+topic name template. The connector supports the following variables: 
+ 
+- ``db``: The database name from the matching namespace.
+- ``sep``: The value of the ``topic.separator`` configuration property.
+  To learn more about this property, see :ref:`source-configuration-kafka-topic`.
+- ``coll``: The collection name from the matching namespace, or an empty
+  string if there is no collection name.
+- ``sep_coll``: The value of the ``coll`` variable prefixed with the value
+  of the ``sep`` variable, or an empty string if the value of ``coll`` is empty.
+- ``coll_sep``: The value of the ``coll`` variable suffixed with the
+  value of the ``sep`` variable, or an empty string if the value of ``coll`` is empty.
+- ``sep_coll_sep``: The value of the ``coll`` variable prefixed and
+  suffixed with the value of the ``sep`` variable, or an empty string if
+  the value of ``coll`` is empty.
+ 
+If you use any of the previous variables in your topic name template,
+you must enclose them in curly brackets (``{}``) for the connector to
+expand them. You can't use curly brackets in the topic name template for
+any other purpose.
+
+.. note:: Escaping Characters
+
+   If your namespace pattern includes characters that need escaping,
+   you must prefix these characters with two backslashes (``\\``) in the
+   pattern. For example, to match a period (``'.'``) character, you must
+   write it as ``'\\.'``.
+ 
+The connector matches namespaces in the following order:
+ 
+1. Pairs with database and collection names in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Pairs with only a database name in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Regex pairs in order.
+#. The wildcard pair. To learn more about this namespace pattern, see
+   the :ref:`Topic Namespace Map with Wildcard Example
+   <topic-naming-namespace-map-wildcard-example>`.
+
+This example shows how to specify the ``topic.namespace.map``
+setting so that the connector performs the following mapping:
+
+- Writes events from databases that match a regular expression to the
+  ``industrial{sep_coll}`` topic. The regex pattern matches any database
+  names that include the string ``'vertical'``, even if the database
+  name has other characters.
```

Comment:
The regex does not match namespaces with database names that are anything but `vertical` exactly. The original text describes what the regex matches as follows (replace `myDb` with `vertical`):

```
The regex namespace pattern matches any namespace with the 'myDb' database name.
```

Reply:
I did not review the examples and their descriptions further. Is it an option to take the examples from the originally proposed text, replace `myDb`, `myColl`, `topicOne`, ... with something meaningful, if this deemed necessary, but otherwise leave the structure, the descriptions and explanations virtually unchanged? Those examples have already been analyzed, tested, and their descriptions are much more likely to be correct than new descriptions written for new tests, or even for the same tests.

Reply:
thanks for pointing it out that I did not interpret the regex correctly. I will use the example from the ticket instead and just change the names. I thought that I had mostly done that in this draft, with the exception of leaving out the wildcard as it is described in the next example.